src/api/models/mermaid.py

```diff
@@ -42,9 +42,11 @@
 FISHBELT_PROTOCOL = "fishbelt"
 HABITATCOMPLEXITY_PROTOCOL = "habitatcomplexity"
 BLEACHINGQC_PROTOCOL = "bleachingqc"
+BENTHIC_TRANSECT_PHOTO_QUADRAT_PROTOCOL = "benthictpq"
 
 PROTOCOL_MAP = {
     BENTHICLIT_PROTOCOL: "Benthic LIT",
+    BENTHIC_TRANSECT_PHOTO_QUADRAT_PROTOCOL: "Benthic Transect Photo Quadrat",
```

Comment:
TBD -- in reading the paper and looking at the mockup more, I'm thinking that the proper name for this should be "benthic photo quadrat transect" to align both with our models -- it inherits from `BenthicTransect` -- and with the concept, which is of a transect where you take photos along the way. It would be our third type of benthic transect:

1. benthic Point Intercept Transect
2. benthic Line Intercept Transect
3. benthic Photo Quadrat Transect